<launch>

    <!-- <include file="$(find dual_ubiquitous_display_bringup)/launch/uvc_camera.launch"/>
    <include file="$(find scan_image_ros)/launch/imshow_distort_fisheye_camera.launch"/> -->

    <node name="detect_person_yolo_scan" pkg="scan_image_ros" type="detect_person_openvino_scan.py" output="screen">
        <param name="scan_topic" value="/scan" />
        <param name="image_topic" value="/camera/color/image_raw" />
        <param name="calib_file" value="$(find scan_image_ros)/data/calibration_result.txt" />
        <param name="config_file" value="$(find scan_image_ros)/config/config_HD.yaml" />
        <param name="laser_point_radius" type="int" value="3" />    <!-- Reprojected laser point radius size in pixels -->
        <param name="time_diff" type="double" value="1" />  <!-- Maximum time difference in seconds between image and laser scan to be registered and processed -->
    </node>

</launch>
